README.md

```diff
@@ -457,3 +457,16 @@ test$.subscribe((result) => {
 // Logs the following:
 // Got number: 5
 ```
+
+## Publishing the package
+
+The package is published manually right now.
+
+Steps to publish:
+
+1. Bump the version in `package.json` and `src/package.json` as needed
```

Comment:
Can you clarify what the differences between these `package.json` are? I honestly don't know why is there a case for having both so if I wanted to produce a change I would have no idea which one to update 🙈

Reply:
`package.json` is for local use, it contains the build steps etc. It's not shipped so the version is probably not important. `src/package.json` is what's actually shipped to NPM, it contains the package metadata, dependencies etc.